### ISSUE TITLE
[Bug]:  databaseURL required on firebase config file

### ISSUE THREAD

**Issue by memins** [we-dance/platform#62]
### What happened?

when you go through the setup for the project you have to put the firebase config in env file but for me, i have to put databaseURL but that was not in firebase config file

### Relevant log output

```shell
ERROR  Missing or incomplete config for current environment 'development': key 'databaseURL' is required!
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by razbakov** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/WeDance-61e70d8b462b5400201aab2d/issues/we-dance/platform/62?planning-poker) with ZenHub @amanuela97 @nelson2411 @Stefanacef

**Comment by razbakov** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/WeDance-61e70d8b462b5400201aab2d/issues/we-dance/platform/62?planning-poker) with ZenHub @harshbrahmxatriya @larrycoal @nelson2411 @Stefanacef @amanuela97

**Comment by razbakov** [comment]
Will be solved in #123
